README.md

```diff
@@ -269,43 +266,41 @@ Be aware that this needs to be tested further.
 
 This is an example of how to mount the windows docker pipe using CLI:
 ```
-docker run --rm -it -p 2015:2015 -v //./pipe/docker_engine://./pipe/docker_engine lucaslorentz/caddy-docker-proxy:ci-nanoserver-1803 -agree -email email@example.com -log stdout
+docker run --rm -it -p 2019:2019 -v //./pipe/docker_engine://./pipe/docker_engine lucaslorentz/caddy-docker-proxy:ci-nanoserver-1803 -agree -email email@example.com -log stdout
 ```
 
 ## Caddy CLI
-All flags and environment variables supported by Caddy CLI are also supported:
-https://caddyserver.com/docs/cli
-
-Check **examples** folder to see how to set them on a docker compose file.
-
-This plugin provides these flags:
+This plugin extends caddy cli with command `caddy docker-proxy` and flags.
 
+Run `caddy docker-proxy --help` to see all available flags:
 ```
--docker-label-prefix string
-      Prefix for Docker labels (default "caddy")
--docker-caddyfile-path string
-      Path to a default CaddyFile (default "")
--docker-polling-interval duration
-      Interval caddy should manually check docker for a new caddyfile (default 30s)
--proxy-service-tasks
-      Proxy to service tasks instead of service load balancer (default false)
--docker-validate-network
-      Validates if caddy container and target are in same network (default true)
+Usage of docker-proxy:
+  -caddyfile-path string
+    	Path to a base CaddyFile that will be extended with docker sites
+  -label-prefix string
+    	Prefix for Docker labels (default "caddy")
+  -polling-interval duration
+    	Interval caddy should manually check docker for a new caddyfile (default 30s)
+  -process-caddyfile
+    	Process Caddyfile before loading it, removing invalid servers
+  -proxy-service-tasks
+    	Proxy to service tasks instead of service load balancer
```

Comment:
This is probably just because of my inexperience with swarm, but what's the difference?

Reply:
When you create a service in swarm, it creates a Virtual IP with a load balancer that balances to all replicas. Tasks are containers executing in order to fulfill a specific service replica configuration.

By default, service VIP will be registered as upstream in Caddy, thus Caddy will be doing load balancing, only routing.

If you chose to proxy to tasks, all running instances of a service are registered in Caddy, and VIP is skipped, thus Caddy load balancer and health checks are used.

Proxying to VIP was more stable and was the first implemented option. Maybe we should change the default to proxy to tasks now.

Reply:
Interesting! Thanks for the explanation.

BTW, Caddy v2 supports SRV DNS as well, I don't think it matters in this case but thought I'd mention it, different way to do the same thing.

Reply:
I noticed that. Unfortunately, Docker doesn't create SRV records for internal DNS. AFAIK.